static/app/views/performance/onboarding/modulesOnboarding.tsx

```diff
@@ -1,6 +1,6 @@
 import styled from '@emotion/styled';
 
-import emptyStateImg from 'sentry-images/spot/performance-waiting-for-span.svg';
```

Comment:
We should delete the old image if its no longer being used

Reply:
Done.